packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 * Fixed
   * Fixed font size in `terra-theme-provider` examples.
+  * Fixed label value when we have 12 hours time format in `terra-time-input` examples.
```

Comment:
Update with main. This should be the only added entry

Reply:
Updated